2021-12-15 03:11:24,104 INFO    MainThread:16684 [wandb_setup.py:_flush():69] setting env: {}
2021-12-15 03:11:24,104 INFO    MainThread:16684 [wandb_setup.py:_flush():69] setting login settings: {}
2021-12-15 03:11:24,104 INFO    MainThread:16684 [wandb_init.py:_log_setup():336] Logging user logs to /Users/tatsukichi/InvertibleTargetPropagation/code/wandb/run-20211215_031124-1ub1zg18/logs/debug.log
2021-12-15 03:11:24,104 INFO    MainThread:16684 [wandb_init.py:_log_setup():337] Logging internal logs to /Users/tatsukichi/InvertibleTargetPropagation/code/wandb/run-20211215_031124-1ub1zg18/logs/debug-internal.log
2021-12-15 03:11:24,104 INFO    MainThread:16684 [wandb_init.py:init():369] calling init triggers
2021-12-15 03:11:24,104 INFO    MainThread:16684 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-12-15 03:11:24,105 INFO    MainThread:16684 [wandb_init.py:init():418] starting backend
2021-12-15 03:11:24,105 INFO    MainThread:16684 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-12-15 03:11:24,114 INFO    MainThread:16684 [backend.py:ensure_launched():132] starting backend process...
2021-12-15 03:11:24,120 INFO    MainThread:16684 [backend.py:ensure_launched():137] started backend process with pid: 16694
2021-12-15 03:11:24,122 INFO    MainThread:16684 [wandb_init.py:init():423] backend started and connected
2021-12-15 03:11:24,125 INFO    MainThread:16684 [wandb_init.py:init():465] updated telemetry
2021-12-15 03:11:24,126 INFO    MainThread:16684 [wandb_init.py:init():484] communicating current version
2021-12-15 03:11:25,042 INFO    MainThread:16684 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.12.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-12-15 03:11:25,042 INFO    MainThread:16684 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-12-15 03:11:25,268 INFO    MainThread:16684 [wandb_init.py:init():522] starting run threads in backend
2021-12-15 03:11:27,907 WARNING MainThread:16684 [wandb_init.py:init():764] interrupted
Traceback (most recent call last):
  File "/opt/anaconda3/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 744, in init
    run = wi.init()
  File "/opt/anaconda3/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 525, in init
    _ = backend.interface.communicate_run_start(run_obj)
  File "/opt/anaconda3/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 799, in communicate_run_start
    result = self._communicate(rec)
  File "/opt/anaconda3/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 537, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/opt/anaconda3/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 82, in get
    is_set = self._object_ready.wait(timeout)
  File "/opt/anaconda3/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/opt/anaconda3/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
